apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html

```diff
@@ -87,6 +108,64 @@ <h1>{{log.operationType}}
                 <p>{{log.log.description}}</p>
             </div> -->
         </div>
+        <!-- stocks -->
+        <div class="address-row-wrapper d-flex" *ngIf="log?.details?.purchases?.account && log?.details?.sales?.account">
+            <div class="address-row mt-05" *ngIf="log?.details?.purchases?.account">
+                <div class="single-address-row">
+                    <span>
+                        Purchase:-
```

Comment:
Do add all the new labels in locale file

Reply:
We have to add new more labels after all work done then we will add